### PR TITLE
wrap lines at 79 characters

### LINKED ISSUE #33: wrap lines at 79 characters
The GitHub diff interface wraps lines at 80 characters. Since we use 100 characters as the max width, this makes for ugly diffs with lots of continued lines. To improve the clarity of these diffs, as well as readability in terminal text editors, we should back away from this edge, and wrap our content at 79 characters.

This is easily done with emacs, but the change would touch almost every file in the repository, so this should be coordinated with other efforts to minimize merge conflicts and the number of branch rebases to be done.

### PR DESCRIPTION
**Pull request by tkphd**
Fixes #33.

### CODE REVIEW

File: _episodes/11-snakemake-intro.md [hunk 1]
```diff
@@ -89,7 +91,7 @@ number of words in the text file.
 
 We can do the same thing for a different book:
 
-```bash
+```
 $ python wordcount.py books/abyss.txt abyss.dat
 $ head -5 abyss.dat
 ```
```
Comment (mikerenfro) [inline _episodes/11-snakemake-intro.md:97]: Do we need a `{: .language-bash}` here?

File: _episodes/13-wildcards.md [hunk 2]
```diff
@@ -169,7 +173,7 @@ Finished job 0.
 >
 > What will happen if you now execute:
 >
-> ```bash
+> ```
 > $ touch *.dat
 > $ snakemake results.txt
 > ```
```
Comment (mikerenfro) [inline _episodes/13-wildcards.md:179]: Missing `{: .language-bash}`?

File: _episodes/13-wildcards.md [hunk 3]
```diff
@@ -84,34 +87,35 @@ rule zipf_test:
...
+rule'. Again, when Snakemake is run it will replace this variable with the
+dependencies.
 
 Let's update our text files and re-run our rule:
 
-```bash
+```
 $ touch books/*.txt
 $ snakemake results.txt
 ```
```
Comment (mikerenfro) [inline _episodes/13-wildcards.md:121]: Missing `{: .language-bash}`?

File: _episodes/13-wildcards.md [hunk 4]
```diff
@@ -183,12 +187,12 @@ Finished job 0.
...
+> > The rules for `*.dat` are not executed because their corresponding
+> > `.txt` files haven't been modified.
 > >
 > > If you run:
 > >
-> > ```bash
+> > ```
 > > $ touch books/*.txt
 > > $ snakemake results.txt
 > > ```
```
Comment (mikerenfro) [inline _episodes/13-wildcards.md:198]: Missing `{: .language-bash}`?

File: _episodes/13-wildcards.md [hunk 5]
```diff
@@ -245,7 +250,7 @@ rule count_words_abyss:
 
 Let's mark `wordcount.py` as updated, and re-run the pipeline.
 
-```bash
+```
 $ touch wordcount.py
 $ snakemake
 ```
```
Comment (mikerenfro) [inline _episodes/13-wildcards.md:256]: Missing `{: .language-bash}`?

**Review by mikerenfro — CHANGES_REQUESTED** [review]

File: _episodes/11-snakemake-intro.md (hunk 1 above)
Comment (tkphd) [reply _episodes/11-snakemake-intro.md:97]: Yes-- it's at the end of the block, not the beginning. This ` ```bash ` tag was severely confusing the lesson-check script.

File: _episodes/13-wildcards.md (hunk 2 above)
Comment (tkphd) [reply _episodes/13-wildcards.md:179]: No, it's moved to the end of the block.

File: _episodes/13-wildcards.md (hunk 4 above)
Comment (tkphd) [reply _episodes/13-wildcards.md:198]: Moved to the end of the block.

File: _episodes/13-wildcards.md (hunk 5 above)
Comment (tkphd) [reply _episodes/13-wildcards.md:256]: Moved to the end of the block.

File: _episodes/13-wildcards.md (hunk 3 above)
Comment (tkphd) [reply _episodes/13-wildcards.md:121]: Moved to the end of the block.

**Comment by tkphd** [comment]
Thanks for the close scrutiny, @mikerenfro!
This lesson retained the older style of syntax highlighting hints, at the beginning of codeblocks rather than after. Those inner hints are purged by this PR, since our lesson-check script does not like them.

**Review by mikerenfro — APPROVED** [review]

**Comment by tkphd** [comment]
@mikerenfro I added you to the @hpc-carpentry/hpc-python-maintainers team so this could be merged. If this offends you, please forgive my faux pas, and leave that team.

**Pull request merged**